index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Project</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+   <section class="body">
+      <header>
+         <nav class="welcome">Hi there</nav>
+         <img src="images/download-removebg-preview.png" class="navigation_bar" alt="Navigation bar">
+       </header> 
+       <section class="dashboard">
+        <div class="backgr">
+            <img src="images/Ellipse 18.png" alt="Ellipse 18">
+        </div>
+        <h1 class="name">Abednego Kibitok</h1>
+        <h2 class="title">Software Developer</h2>
+        <p class="pa">
+            I can help you build a product ,<br>
+             feature or website Look through<br>
+              some of my work and experience! If<br>
+               you like what you see and have a<br>
+                project you need coded,  don’t<br>
+                 hestiate to contact me.
+        </p>
+        <img src="images/Vector 4.png" class="line" alt="Horizontal line">
+        <!--<h1 class="name">LET'S CONNECT</h1>
+        <br>
+        <ul>
+            <li><a href="#"><img src="images/Vector.png" alt="Github_icon"></a></li>
+            <li><a href="#"><img src="images/Group.png" alt="Linkedin_icon"></a></li>
+            <li><a href="#"><img src="images/VectorX.png" alt="X_icon"></a></li>
+            <li><a href="#"><img src="images/Teiitter.png" alt="Twitter_icon"></a></li>
+            <li><a href="#"><img src="images/Vectorm.png" alt="M_icon"></a></li>
+        </ul>-->
```

Comment:
- [ ] Kindly remove this commented code, this will make your codebase cleaner. 👍